test/loader.test.js

```diff
@@ -57,6 +63,12 @@ it.describe("gofigure.Loader", function (it) {
             assert.deepEqual(loader.config, helper.allDeepMerge());
         });
 
+        /* it.should("load from etcd", function () {
```

Comment:
Not yet.  I'm working on making the tests run with nock now that they work against a running etcd server.  I should have working tests sometime in the next couple of hours.